superflore/generators/ebuild/gen_packages.py

```diff
@@ -30,6 +30,13 @@
 org = "Open Source Robotics Foundation"
 org_license = "BSD"
 
+"""
+TODO(allenh1): This is a blacklist of things that
```

Comment:
Triple quotes aren't comments they're multi line strings that are evaluated by the interpreter.